internal/internal_task_handlers.go

```diff
@@ -633,6 +633,7 @@ func (wth *workflowTaskHandlerImpl) createWorkflowContext(task *workflowservice.
 		Namespace:                wth.namespace,
 		Attempt:                  attributes.GetAttempt(),
 		lastCompletionResult:     attributes.LastCompletionResult,
+		lastFailure:              attributes.ContinuedFailure,
```

Comment:
I have one conceptual concern here. We don't expose `Failure` from SDK now. We expose go `error`s. Shouldn't you follow same pattern and convert `Failure` to `error`. There is func for this already. @mfateev, what do you think about it?

Reply:
Yes, I missed it. Let's convert it to the error, failure is internal to the framework.

Reply:
Fixed. Thanks guys!